bin/cli.js

```diff
@@ -152,6 +156,7 @@ const getConfig = args => {
     prioritizedEntries: [],
     private: false,
     contentsTable: 0,
+    mergeTitleDescription: true,
```

Comment:
I'd prefer separateTitleDescription to have all options as falsy by default since we are making this behavior as default.